client/src/components/ProgressBar/progress-bar.tsx

```diff
@@ -84,4 +82,4 @@ function ProgressBar({
 
 ProgressBar.displayName = 'ProgressBar';
 
-export default connect(mapStateToProps)(withTranslation()(ProgressBar));
+export default ProgressBar;
```

Comment:
If I am not missing something, losing the connection to redux store forces you to add props, which you did, but you need to add it every time the component is added.

you can:
- connect it back to redux store, and remove the props.
- add the props to every place that use progress bar.

Reply:
Hey @Sboonny,

Thanks for reaching out, please follow this comment:

https://github.com/freeCodeCamp/freeCodeCamp/pull/49395#issuecomment-1499989523